front/app/containers/ProjectsShowPage/shared/survey/index.tsx

```diff
@@ -252,6 +252,7 @@ class Survey extends PureComponent<Props, State> {
                 typeformUrl={surveyEmbedUrl}
                 email={email || null}
                 user_id={user_id}
+                language={language || undefined}
```

Comment:
You're already defining undefined as a fallback in the `language` const, so this has no added value. The email prop above has the same 'issue', the fallback to `null `is not needed.

Reply:
When I remove the " || null " from the email prop, I receive the following error:
![error](https://user-images.githubusercontent.com/33987955/155503485-2b06c8e1-982b-41ae-9744-c75099e78ff1.png)
It seems like it's required to check that it will not be undefined. Do you have any suggestions?

Reply:
I think I'll merge this for now, but add the refactor to my to-do list.

Reply:
@amanda-anderson you can merge for now indeed and do the refactoring later. Btw, you need to tag me in these comments, I don't get notifications.

Reply:
Also, if you're stuck with that refactoring, happy to help you out.